### DISCUSSION TITLE
Taskbar Clock Customisation Not Working

### DISCUSSION THREAD

**Discussion by AlanWade** [ramensoftware/windhawk-mods#684] [Q&A] [answered]
Does anybody know if there are plans to update the above mod? it no longer works with the latest insider release of Win 11
Thanks for reading.

**Comment by m417z** [comment, marked as answer]
Due to a Microsoft maintenance, some symbol files are unavailable. As a result, some mods temporary don't work on insider builds.
For details, see https://github.com/ramensoftware/windhawk-mods/issues/685.

**Reply by AlanWade** [reply]
Thanks for the feedback, hopefully Microsoft will fix it soon.
